docs/ConfigurationFile.md

```diff
@@ -233,7 +233,7 @@ exports.config = {
         source: true,       // <boolean> hide source URIs
         profile: [],        // <string[]> (name) specify the profile to use
         strict: false,      // <boolean> fail if there are any undefined or pending steps
-        tagExpression: [],  // <string[]> (expression) only execute the features or scenarios with tags matching the expression
+        tagExpression: '',  // <string> (expression) only execute the features or scenarios with tags matching the expression
```

Comment:
Can you search this line of code in the whole repo, I am afraid we have more occurrences of this everywhere that need an update.

Reply:
The other occurrences already refer to string:
![image](https://user-images.githubusercontent.com/15637471/91337973-33eb5380-e7a2-11ea-9663-40e8ee3e027c.png)

Reply:
Ah ok, I just found we should update this line: https://github.com/webdriverio/webdriverio/blob/master/examples/wdio.conf.js#L177